openedx/features/edly/models.py

```diff
@@ -35,3 +37,22 @@ class Meta:
 
     def __str__(self):
         return '{name}: ({slug})'.format(name=self.name, slug=self.slug)
+
+
+class EdlyUserProfile(models.Model):
+    """
+    User profile model for Edly users.
+    """
+    user = models.OneToOneField(User, unique=True, db_index=True, related_name='edly_profile', on_delete=models.CASCADE)
+    edly_sub_organizations = models.ManyToManyField(EdlySubOrganization)
+
+    @property
+    def get_linked_edly_sub_organizations(self):
+        """
+        Helper method to get list of slugs of edly sub organizations of a user.
+
+        Returns:
+            List of edly sub organizations slugs
```

Comment:
Need to mention `Returns` type first.
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings

Reply:
Done